index.html

```diff
@@ -0,0 +1,130 @@
+<!DOCTYPE html>
+
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <link href="mystyle.css" rel="stylesheet" type="text/css">
+
+    <title>Welcome to Erik's Site</title>
+</head>
+
+<body>
+    <div class="container">
+        <div class="“col-md-12”">
+            <div id="header">
+                <h1></h1>
+
+                <h1>Erik Biberstein </h1><br>
+
+                <h2></h2>
+
+                <h2><i>Student at The Ohio State University</i> </h2><br>
+
+                <a href = "#activities"> Activities </a>
+                <hr>
+            </div>
+
+            <div id="intro">
+                <div class = "col-md-12">
+                <i></i>
+
+                <p style="font-style: italic">Hello there! My name is Erik
+                Biberstein and I am a sophomore Mechanical Engineer at The Ohio
+                State University. I am very interested in coding and I have
+                built this website to showcase some of my work</p><i></i>
+
+                <p style="font-style: italic">I am currently seeking a mechanical
+                engineering/computer science Summer co op or internship for
+                2016</p>
+
+                <p style="font-style: italic">If you would like to learn more
+                about me check out my <a href=
+                "http://github.com/eBibby">Github</a> and <a href=
+                "http://www.linkedin.com/pub/erik-biberstein/103/4b1/543">LinkedIn</a>
+                pages</p><i></i>
+            </div>
+        </div>
+
+            <div id="personal_pic" style="font-style: italic"><img alt=
+            "A picture of myself" height="300" src="PersonalPicture.jpg" width=
+            "300"></div><i></i>
+
+            <div id="activities">
+                <i></i>
+
+                <h3 style="font-style: italic">Activities</h3><i></i>
+
+                <ul>
+                    <li><i>Humanitarian Engineering Scholar's Program</i></li>
+
+                    <li style="list-style: none">
+                        <i></i>
+
+                        <ul>
+                            <li><i>Engineering community centered around
+                            humanitarian engineering principles; critically
+                            solving problems while considering cultural
+                            implications</i></li>
+
+                            <li style="list-style: none"><i></i></li>
+                        </ul>
+                    </li>
+
+                    <li>
+                        <i>Security Operations Specialist</i>
+
+                        <ul>
+                            <li><i>I assist security team in building firewalls
+                            to be used in classrooms across campus</i></li>
+
+                            <li style="list-style: none"><i></i></li>
```

Comment:
I see you were trying to get rid of those pesky bullet points. Yeah, they're not great. The thing is - this style belongs on the `ul` tag instead of the `li` tags.

This is going a little into the future, but there's other bullet point styles you can try as well. Here's some examples on those: https://css-tricks.com/almanac/properties/l/list-style/
To use these, try something like `<ul style="list-style: square outside none">`